### PR TITLE
Drop entities on joints

### PR DESCRIPTION
**Pull request by sethalves**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: c8cecad537a014b5551c816f5cd6387b46f00555 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: c8cecad537a014b5551c816f5cd6387b46f00555 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 90fe5582a8093d2dbf218fd7703c1b64357c10f5 available at: https://gist.github.com/332004d6fe2d96148a7d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 372f6d12c8511d16c76d42f32017a989ae2f194b available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

File: libraries/entities/src/EntityItemProperties.cpp [hunk 1]
```diff
@@ -374,9 +374,7 @@ QScriptValue EntityItemProperties::copyToScriptValue(QScriptEngine* engine, bool
         COPY_PROPERTY_TO_QSCRIPTVALUE(PROP_JOINT_TRANSLATIONS, jointTranslations);
     }
 
-    if (_type == EntityTypes::Model || _type == EntityTypes::Zone || _type == EntityTypes::ParticleEffect) {
-        COPY_PROPERTY_TO_QSCRIPTVALUE_GETTER(PROP_SHAPE_TYPE, shapeType, getShapeTypeAsString());
-    }
+    COPY_PROPERTY_TO_QSCRIPTVALUE_GETTER(PROP_SHAPE_TYPE, shapeType, getShapeTypeAsString());
```
Comment (ZappoMan) [inline libraries/entities/src/EntityItemProperties.cpp]: hmmmm -- whats this? are you really adding shape type to ALL entities?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7d5e90f871cba56e3289ee8682b7546560f27ccf available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

File: libraries/entities/src/EntityItemProperties.cpp (hunk 1 above)
Comment (sethalves) [inline libraries/entities/src/EntityItemProperties.cpp]: Yeah that probably wasn't right.  I'm trying to make it so JS can know if something has a shape or not (and therefore know if something has a rigid-body in bullet or not).

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2ce623f4288a9f2b873fd9faa2f597946309182f available at: https://gist.github.com/e16820db2fe8493efcd7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2ce623f4288a9f2b873fd9faa2f597946309182f available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 893d5cb5ff79180060075fd1cddda2bbeb6a9ad0 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 37ae0c732be2084c43f2c4153b852786ab8f0286 available at: https://gist.github.com/e16820db2fe8493efcd7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 16b53967b88ded75c5da468f74813e9b35ac4c46 available at: https://gist.github.com/eaf1d5dc8220ba50e663<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d0cb8aa3e175adea6bf1b0a75ec28804aa106aec available at: https://gist.github.com/e608abf83758678dbc9e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 16b53967b88ded75c5da468f74813e9b35ac4c46 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0db7afb958b16f07e68761077d62e3ee53212e24 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d926b7adf03409cf48650ce935d20e068063b5d1 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ecac03283a8aa4cfb9244ab5fdbf84dd237d9f33 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 37d116b51ac95572402e44043928a4dbb0a2be8f available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: dd80b98cf4393b850c36d207aadf06314f4e4a46 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8039c941caad75184f8cabcdd0c40c4e87f2b15b available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f87abe446bba8106bc26b9974d6957e7df1b9be4 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d9cc3feee67640968541c990bd90d725c6178b9d available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9e0f3954de4b3fd2d9655b3eab0bafe418cd7fe5 available at: https://gist.github.com/332004d6fe2d96148a7d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 43da258459586076e55a1d0eb99e77762254414f available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 574822a290c4e9ed28174f09ca78c7797d77f4db available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5377d0d97b75f10673803fc2a02dd55fc8d99ebe available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 287df0dd76919c411d57b08a2c09c12914c15953 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a18ba6793a15594cc13cc9366353b7774b5280c0 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc2762815e8484864184c74f0006a0a5f5a2a8c3 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 49ca35e28ea7f445c30b5ac59390b0e401e09f22 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e74350fcfd9dc99716b841e2e04430947a3f69c3 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2b84baf529acb5d9f790b3f8942a7d06866fc4b3 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c9e22a11fd03976cd851f31ea23612e1d0d7f344 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 051e9b6fdd7c1cd94443afad15051e2fe9d274b3 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f3feeedefb044f8ed1fcea474b38dde0d1c5aad0 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c2640615ef4e245442d4a4052ab294f45dcd990b available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e7efe2c2ace1db1153920c5b41d43526993a2a67 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc95f7bbfaa990c942580bd6485d43df38efede2 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e57e2e8496b52d9376ccc7be1f5e5b165659d27c available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7645ebf2c0d1863f648bfca8ba1be838dde3833d available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4f8c126ce504ca1b58843e8875edda8a683080b5 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 80e4674e260113721295cc684508e67c2dff6fba available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: befd5e3df2e9af6254025c534c3b4a428d18cdc4 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 42761c604855f1dab05add2f58eb70768b8208f5 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 62cf8dcc08d29c8f8ca5c900d87671818603200f available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9c3fff216b533635175fbb50b3d8051d9658918c available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d4b2b6dbbb71783a783682dc27fbe145ff74e45d available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0b4aae44749129ec8dfdc90413bfea8546a457bf available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9eb7d3ddd2fe2201acdd9226e1cd095cc2658c05 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0c1c656120db81ef782ee9a63ed8d152a6f06772 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8fb5bee22691e964c4a131d28ce3253aec0bd3bc available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4b53196cb598378aaf5aac2eb448076c3e0d67b7 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2cb85dca3206ec75724fa69778e3db3cebe6f127 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 29798717611ac9f6d0514d82e7836412734f7052 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9735188729304ecc7592da172f99b1e68b7ddf42 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 93ebb70c234cf1432cd1a390114cb33472305423 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6be11dc3ef3ef85177af7cb72ef334540ede142b available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2bf3a7473864f61f5121a0b5a2de5c3d2cab7086 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 164075e056057efdcd88a3befd2521f05f1fbc0f available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fee00546b6a684c042577e333bbdc707fc0c8bd7 available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7e93e68f76798164d87e60d8e806a1ccfada264c available at: https://gist.github.com/e15b9c647a8d495a115e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6784.dmg

**Pull request closed without merging**